test/DocumentFormat.OpenXml.Benchmarks/ElementMetadataTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BenchmarkDotNet.Attributes;
+using DocumentFormat.OpenXml.Framework.Metadata;
+
+namespace DocumentFormat.OpenXml.Benchmarks
+{
+    public class ElementMetadataTests
+    {
+        [GlobalSetup]
+        public void Setup()
+        {
+            _element = new AlternateContent();
+        }
+
+        [Benchmark]
+        public void CreateWithInstance()
+        {
+            _ = ElementMetadata.Create(_element);
```

Comment:
Can you have this return the metadata? You can have it return cast as `object` as it's an internal type if needed. The release builds may optimize away the call if not returned.